src/Tochka.JsonRpc.Client/Services/JsonRpcIdGenerator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Logging;
+using Tochka.JsonRpc.Common.Models.Id;
+
+namespace Tochka.JsonRpc.Client.Services;
+
+public class JsonRpcIdGenerator : IJsonRpcIdGenerator
+{
+    private readonly ILogger<JsonRpcIdGenerator> log;
```

Comment:
nit, this is too much. instal log class for what? One unneeded log?

Reply:
✅